src/events/base.py

```diff
@@ -36,12 +37,13 @@ def get_app_status(
         if not k8s_manager.trusted():
             return Status.NOT_TRUSTED.value
 
-        s3 = s3
+        if s3 and azure_storage:
```

Comment:
> The only major comment would be that it seems that we are not covering for the case where there is both a s3 relation and an azure relation.

> What do we do in this case? I thought that the idea was to go into blocked status, since the two can be conflicting, right?

We have done it here @deusebio.

Reply:
ops. sorry! The code was exactly in the place that I would have expected it to be. Sorry for that! :D 

All good then! I have some small comments (mostly minors and non blocking), but I'm happy with the PR!